packages/openid4vc/src/openid4vc-verifier/__tests__/openid4vc-verifier.test.ts

```diff
@@ -65,7 +65,7 @@ describe('OpenId4VcVerifier', () => {
       expect(jwt.payload.additionalClaims.redirect_uri).toEqual(
         `http://redirect-uri/${openIdVerifier.verifierId}/authorize`
       )
-      expect(jwt.payload.additionalClaims.response_mode).toEqual('post')
+      expect(jwt.payload.additionalClaims.response_mode).toEqual('direct_post')
```

Comment:
this feels like a breaking change. Do older versions of the oidp4v-sip lib already support direct_post?

Reply:
support is there starting from v17